lib/Process.php

```diff
@@ -238,7 +238,7 @@ public function getStdout(): ProcessInputStream {
      */
     public function getStderr(): ProcessInputStream {
         if (!$this->handle) {
-            throw new StatusError("The process has not been started");
+            throw new StatusError("Process has not been started.");
```

Comment:
Why did you change these?

Reply:
I think the exception message need to be unified.
Some place using ```Pocess``` and others using ```The process```.